science/gr-limesdr/Portfile

```diff
@@ -17,12 +17,12 @@ homepage            https://wiki.myriadrf.org/Gr-limesdr_Plugin_for_GNURadio
 subport gr-limesdr-devel {}
 if {[string first "-devel" $subport] > 0} {
 
-    github.setup myriadrf gr-limesdr 47c6d40d369efc68f3f0d19d5a0105cd2d68732b
-    version   20190617
-    checksums rmd160 dd5b0ea9624f2d620ea56fabc7acd7a814f81af1 \
-              sha256 297d7ab8ef50c306ac0cfc12574e1ce7fedfabc8dbde7e2119ca11d77315ed03 \
-              size   3086257
-    revision  0
+    github.setup    myriadrf gr-limesdr 14b29b3b9c46571fe6dbfd4c818ae5db419d1aa7
+    version         20190703-[string range ${github.version} 0 7]
+    checksums       rmd160 dd5b0ea9624f2d620ea56fabc7acd7a814f81af1 \
+                    sha256 297d7ab8ef50c306ac0cfc12574e1ce7fedfabc8dbde7e2119ca11d77315ed03 \
+                    size   3086257
+    revision        0
```

Comment:
It looks like extra spaces or tabs got inserted here into the diffed section?

Reply:
to re-align the lines to the other content

Reply:
@ra1nb0w You don't like my new spacing? I don't really care which way the spacing is, though my current preference is for tighter, as it was before. It's purely a matter of preference, and doesn't impact Portfile functionality.

Reply:
Also: Thank you for changing the `version` ... I really like this new version for devel ports much better than just the date.

Reply:
> @ra1nb0w You don't like my new spacing?

no preference at all; I just aligned like before So if you prefer I can remove the new spaces.
What I personally don't like is the no respect of indent directive for emacs/vim on header for the line after checksum; should be
```
checksums           rmd160  112b7df7bfa4796d6d027d7b374885b81c64e6cf \
    sha256  f54b1e0d467dfd103a119b913d96b7c3a56baf42d0afc8c2bc5c551196b100a3 \
    size    35735991
```

Reply:
Let's leave the spacing tight then. If you want to update this PR that's great. If not I'll get to updating this port shortly.

Reply:
I, too, would love for the indent directive to somehow handle the checksum more robustly! There are a few other places of similar annoyance; small, but sometimes important!

Reply:
> Let's leave the spacing tight then. If you want to update this PR that's great. If not I'll get to updating this port shortly.

done